tests/test_arrays.py

```diff
@@ -0,0 +1,124 @@
+import uproot, uproot_methods
+import numpy as np
+import fnal_column_analysis_tools
+from fnal_column_analysis_tools.arrays import Builder
+
+file = uproot.open("nano_5.root")
```

Comment:
Add this file to `tests/samples` if it isn't too big.  Also, I think the whole test should be wrapped in a descriptive function name.

Reply:
Gotta change the path here too.  Also, 40mb is a bit big, possible to make it a bit smaller, say <1 mb?

Reply:
Path is consistent no? I added the root file in tests.